frr/if_map.c

```diff
@@ -81,13 +81,13 @@ bool add_ifindex_mapping(ifindex_t grout_ifindex, ifindex_t frr_ifindex) {
 	return true;
 }
 
-ifindex_t ifindex_grout_to_frr(int16_t grout_ifindex) {
+ifindex_t ifindex_grout_to_frr(ifindex_t grout_ifindex) {
 	struct ifindex_mapping key = {.grout_ifindex = grout_ifindex};
 	struct ifindex_mapping *found = grout_to_frr_find(&grout_to_frr_mappings, &key);
 	return found ? found->frr_ifindex : IFINDEX_INTERNAL;
 }
 
-uint16_t ifindex_frr_to_grout(ifindex_t frr_ifindex) {
+ifindex_t ifindex_frr_to_grout(ifindex_t frr_ifindex) {
```

Comment:
It looks like this commit isn't needed anymore. The last commit does not create a fake sr0 interface.